lib/use_packs/invalid_pack_name_error.rb

```diff
@@ -0,0 +1,42 @@
+
+module UsePacks
+  class InvalidPackNameError < StandardError
+    class << self
+      def configured_pack_locations
+        @configured_pack_locations ||= Packs.config.pack_paths
+      end
+
+      def permitted_pack_locations
+        return @permitted_pack_locations if defined?(@permitted_pack_locations)
+
+        if configured_pack_locations == %w[packs/* packs/*/*]
```

Comment:
I'm guessing you're doing this to preserve the existing behavior. That is, if the default value for `pack_paths` is equal to this, use the previous value for permitted pack locations. I think that's a good sentiment, but for simplicity and clarity for now let's just directly use the configured pack locations always and push the user to fully qualify their `pack_paths` in `packs.yml`.